### ISSUE TITLE
Close button on the modals doesn’t work for Windows Narrator and NVDA screen readers

### ISSUE THREAD

**Issue by sraby** [JustFixNYC/who-owns-what#278]
According to the [Accessibility Audit](https://docs.google.com/document/d/1wtyZ81DCZroVavxE0iMnA6oWAH28VPqIJ-TjbirKvYU/edit#), "interacting with it just makes the screen reader say “item has no secondary action” and nothing happens, while in NVDA nothing is narrated and nothing happens."
